views/index.pug

```diff
@@ -68,37 +68,37 @@
   body(style="padding-top: 8rem !important;")
     - const utmValue = utmSource ? "/?utm_source=" + utmSource : '/';
     header
-      nav.navbar.is-fixed-top#navbar(role="navigation" style='background-image: url("/static/imgs/hero-cover-sticky-header.jpg"); background-size: cover; background-repeat: no-repeat; color: white; background-position: center center; z-index: 100; max-height: 120px; height: 120px; padding-left: 1.5rem; border-radius: 0; padding-right: 1.5rem; ')
-        .navbar-brand
-          .column(style="width: auto;")
-            .row
+      nav.navbar.is-fixed-top#navbar(role="navigation" style='background-image: url("/static/imgs/hero-cover-sticky-header.jpg"); height: 17vh; background-size: cover; background-repeat: no-repeat; color: white; background-position: center center; z-index: 100; padding-left: 1.5rem; border-radius: 0; padding-right: 1.5rem; ')
```

Comment:
If we want to do future work on the responsiveness here, it will be important to keep in mind the built-in pixel cutoffs for Bulma referenced [here](https://bulma.io/documentation/overview/responsiveness/)